TestAssets/NonRestoredTestProjects/DotnetAddP2PProjects/App/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NSLib1;
+using NSLib3;
+
+namespace ConsoleApplication
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+#if NETCOREAPP1_0
+            Lib1.HelloNCA();
+            Lib3.HelloNCA();
+#endif
+
+#if NET451
+            Lib1.HelloNet451();
+            Lib3.HelloNet451();
+#endif
+        }
+    }
+}
```

Comment:
I agree that I overcomplicated slightly. Dotnet new unfortunatelly doesn't create projects with target framework and also if you actually have missing dependency and don't use it in a project it can still compile (AFAIR msbuild was treating missing p2ps as warnings so I'm just making sure we do in fact reference the right thing as this won't compile if something is missing - I agree that I overcomplicated it though)